tests/federation_room_join_partial_state_test.go

```diff
@@ -3636,6 +3637,49 @@ func TestPartialStateJoin(t *testing.T) {
 	//   - Join+Leave+Rejoin works
 	//   - Join + remote kick works
 	//   - Join + remote ban works, then cannot rejoin
+
+	t.Run("Purge during resync", func(t *testing.T) {
+		if runtime.Homeserver != runtime.Synapse {
+			// TOOD: Pull this into a Synapse-specific suite when someone figures out how
+			// to do that (https://github.com/matrix-org/complement/issues/226)
+			t.Skipf("Skipping test of Synapse-internal API on %s", runtime.Homeserver)
+		}
+		t.Log("Alice begins a partial join to a room")
+		alice := deployment.RegisterUser(t, "hs1", "t41alice", "secret", true)
```

Comment:
I believe t46 is the next unused user ID.
```suggestion
		alice := deployment.RegisterUser(t, "hs1", "t46alice", "secret", true)
```

Reply:
Oops, thanks. Copy-paste error.